docs/source/conf.py

```diff
@@ -6,20 +6,22 @@
 
 import os
 import sys
+import subprocess
 
 from sphinx.ext.autodoc import between
 from datetime import datetime
 
 
 sys.path.append(os.path.abspath("./_ext"))
 
-for (dir_path,dir_names,file_name) in os.walk('../../', topdown=True): 
+for (dir_path, dir_names, file_name) in os.walk('../../', topdown=True):
```

Comment:
```python
"""
Signature: os.walk(top, topdown=True, onerror=None, followlinks=False)
Docstring:
Directory tree generator.

For each directory in the directory tree rooted at top (including top
itself, but excluding '.' and '..'), yields a 3-tuple

    dirpath, dirnames, filenames

dirpath is a string, the path to the directory.  dirnames is a list of
the names of the subdirectories in dirpath (excluding '.' and '..').
filenames is a list of the names of the non-directory files in dirpath.
Note that the names in the lists are just names, with no path components.
To get a full path (which begins with top) to a file or directory in
dirpath, do os.path.join(dirpath, name).

If optional arg 'topdown' is true or not specified, the triple for a
directory is generated before the triples for any of its subdirectories
(directories are generated top down).  If topdown is false, the triple
for a directory is generated after the triples for all of its
subdirectories (directories are generated bottom up).
...
"""
```

Just the help text from `os.walk`, you could name the variables here as `dirpath`, `dirnames`, and `filenames` or even with their example: `for root, dirs, files in os.walk('python/Lib/email'):`, you could use `root`,`dirs`,`files`. Also, it is not necessary to wrap the variables in parentheses.